ledfx/effects/rain.py

```diff
@@ -27,6 +27,11 @@ class RainAudioEffect(AudioReactiveEffect):
                 description="color for low sounds, ie beats",
                 default="white",
             ): validate_color,
+            vol.Optional(
+                "pulse_strip",
+                description="Pulse the entire strip to the beat",
+                default="Off",
+            ): vol.In(["Off", "Lows", "Mids", "Highs"]),
```

Comment:
This is one of our oldest effects - and I must be honest, the lack of a config_updated function in here confuses me a little but everything is working as expected.

I wonder if assigning to self rather than grabbing the config out of ledfx is faster/more performant, but it'd be interesting to see the work/effort/time/performance trade off for using the config values vs assigning to self for every config value that we add.

Reply:
The difference between variable assignment and key resolution in the config is probably difficult to measure, for non iterative behaviors, but when you get into loops, I expect it to add up quickly.